cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -419,30 +400,12 @@ <h2>
                 </div>
             {% endif %}
             {% if card.monthly_fee %}
-                {% if card.annual_fee %}
+                {% if card.annual_fee or not card.periodic_fee_type %}
```

Comment:
I've already forgotten the intricacies of how the periodic fee data is structured, but I think we can get rid of the `or not card.periodic_fee_type` here and everywhere below. I put it in line 394 because we use the same space we show an annual fee for cards with no periodic fee (we just show it as "$0 annual fee"). But I _think_ `not card.periodic_fee_type` will never be true here since it's nested under an `if card.monthly_fee`, which means `card.periodic_fee_type` should contain at least one item, "Monthly" (in theory, anyway—who knows if there are data errors that prove that assumption wrong).

If I'm misunderstanding that or there's some weird data thing going on that makes the `or not card.periodic_fee_type` necessary, ignore all that!